backend/util/src/prelude.rs

```diff
@@ -0,0 +1,5 @@
+//! Our own prelude that is wildcard imported in every other module. That way,
+//! commonly used symbols are easily available.
+
+pub use anyhow::{anyhow, bail, Context as _, Result};
```

Comment:
So with this and `use tobira_util::prelude::*;`, an unqualified `Result` means `anyhow::Result`, right? Is this what we want? And if so, how would you get the standard `Result` back, if you wanted it?

Reply:
Yes. The anyhow result is defined as:

```
type Result<T, E = anyhow::Error> = std::result::Result<T, E>;
```

So it's just a default parameter and you can still say `Result<u32, MyError>` in our code. Additionally, you can also fully qualify `std::result::Result`.

Reply:
Aah, okay. I'm fine with that.